### DISCUSSION TITLE
Mysql version requirements more explicit

### DISCUSSION THREAD

**Discussion by dlen** [apalis-dev/apalis#259] [General]
Starting with v0.5 mysql backend uses `SKIP LOCK`. I think it could be good for other developers to have more visibility on the README about mysql version requirements. (SKIP LOCK was added in mysql 8.0 if I'm not mistaken)

**Comment by geofmureithi** [comment]
I can add that. 
I am curious, are you using my MySQL?

**Comment by dlen** [comment]
Yes, I have been using your library in version 0.4 and mysql 5.x. I upgraded to 0.5 and found a blocker with the SKIP LOCK but took me a while to realize :sweat_smile:

**Comment by geofmureithi** [comment]
@dlen could you help with https://github.com/geofmureithi/apalis/issues/48? I don't really use MYSQL but this issue seems to be persistent.
